vector/src/main/res/values-fr/strings.xml

```diff
@@ -496,7 +498,7 @@ Vous pouvez ajouter votre adresse e-mail à votre profil dans les paramètres.</
     <string name="auth_reset_password_success_message">Votre mot de passe a été réinitialisé.
 
 Vous avez été déconnecté de tous les appareils et ne recevrez plus de notifications. Pour réactiver les notifications, reconnectez-vous sur chaque appareil.</string>
-
+    <string name="start_message">\n\n\n\nBienvenue sur Tchap.</string>
```

Comment:
The textview position should be updated instead of adding several `\n``
BTW, this string should not be ended with a `.`